src/main/java/edu/kpi/testcourse/storage/UrlRepositoryFakeImpl.java

```diff
@@ -2,14 +2,17 @@
 
 import edu.kpi.testcourse.entities.UrlAlias;
 import java.util.HashMap;
+import java.util.LinkedList;
 import java.util.List;
 import javax.annotation.Nullable;
 
 /**
  * An in-memory fake implementation of {@link UrlRepository}.
  */
 public class UrlRepositoryFakeImpl implements UrlRepository {
-  private final HashMap<String, UrlAlias> aliases = new HashMap<>();
+
+  //Make it public to make my testing process less painful
+  public final HashMap<String, UrlAlias> aliases = new HashMap<>();
```

Comment:
You cannot do this! You should use only public API in the tests. Instead of a hash map field, use in tests methods like findUrlAlias.